.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "gdal-src/source"]
```

Comment:
Should we point this to a specific tag? I'm not sure we want to bundle `master`.